data/recommendation/notify.js

```diff
@@ -131,29 +130,25 @@ class Notify {
   showWhyBox() {
     this.whyBoxShowing = true;
     const lastRec = document.querySelector('.addon-box:last-child');
-    lastRec.className = 'hiddenRec';
-    lastRec.setAttribute('hidden', 'true');
+    lastRec.classList.add('hidden');
     const whyBox = document.querySelector('#why-box');
-    whyBox.removeAttribute('hidden');
-    whyBox.className = 'addon-box';
+    whyBox.classList.remove('hidden');
     const whyButton = document.querySelector('#why-button');
-    whyButton.className = 'grayed';
+    whyButton.classList.add('grayed');
   }
 
   closeWhyBox() {
     this.hideWhyBox();
-    const lastRec = document.querySelector('.hiddenRec');
-    lastRec.className = 'addon-box';
-    lastRec.removeAttribute('hidden');
+    const lastRec = document.querySelector('.addon-box:last-child');
```

Comment:
Do you mean that the user should still be able to see the last recommendation in the list when the why box is shown? Instead of having the why box replace the last recommendation in the UI?